cloud/amazon/route53_health_check.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: route53_health_check
+short_description: add or delete health-checks in Amazons Route53 DNS service
+description:
+     - Creates and deletes DNS Health checks in Amazons Route53 service
+     - Only the port, resource_path, string_match and request_interval can be updated
+version_added: "2.0"
+options:
+  state:
+    description:
+      - Specifies the action to take.
+    required: true
+    choices: [ 'present', 'absent' ]
+  ip_address:
```

Comment:
Actually this is an error in the documentation again. Fixed in f7cfa00d6f9b2aeed70bfcfb5df6b6ef2870de4e .
If neither the fqdn nor the ip_address are provided then AWS will return a descriptive error. I suppose I could add a check and build my own error response.

I am currently using the module in the following forms, so I am sure those are working. Again I wish we could build unit tests or integration tests:
```yaml
- local_action:
    module: route53_health_check
    state: "{{ binding_state }}"
    ip_address: "{{ ansible_ec2_public_ipv4 }}"
    type: TCP
    port: 5443
    request_interval: 10
    failure_threshold: 2

- local_action:
    module: route53_health_check
    state: "{{ binding_state }}"
    fqdn: "api-{{ cluster_name }}.{{ cluster_domain }}"
    ip_address: "{{ ansible_ec2_public_ipv4 }}"
    type: HTTP
    resource_path: /internal/info
    request_interval: 10
    failure_threshold: 2
```
For auth I am using the AWS_SECRET_ACCESS_KEY and AWS_ACCESS_KEY_ID environment variables. What is your method of auth ?